src/test/appSpec.js

```diff
@@ -1,14 +1,76 @@
 var server = require('../app.js')
 var configUtil = require('../libs/sb-config-util')
 var _ = require('underscore')
-
+var fs = require('fs')
 var request = require('request')
 var host = 'http://localhost:5000'
 const whiteListedChannelList = process.env.sunbird_content_service_whitelisted_channels
 const blackListedChannelList = process.env.sunbird_content_service_blacklisted_channels
-var whiteListQuery = whiteListedChannelList ? whiteListedChannelList.split(',') : []
-var blackListedChannelListNew = blackListedChannelList ? blackListedChannelList.split(',') : []
-var blackListQuery = {'ne': (blackListedChannelListNew)}
+
```

Comment:
this Spec file has to be refactored with the changes done please fix this and please don't comment any old specs